app/views/layouts/_navbar.html.erb

```diff
@@ -1,95 +1,91 @@
-<% if user_signed_in? %>
-  <div class="top-bar-container" data-sticky-container>
-    <div class="sticky" data-sticky data-options="anchor: page; marginTop: 0; stickyOn: small;">
-      <div class="title-bar" data-responsive-toggle="example-menu" data-hide-for="medium">
-        <button class="menu-icon" type="button" data-toggle="example-menu"></button>
-        <div class="title-bar-title">
-          <% if current_organization.present?  %>
-            <%= current_organization.name %>
-          <% elsif current_user.organization.present?  %>
-            <%= current_user.organization.name %>
-          <% else %>
-            <span>DiaperBase</span>
-          <% end %>
-        </div>
-      </div>
-      <div class="top-bar" id="example-menu">
-        <div class="top-bar-left">
-          <ul class="dropdown menu" data-dropdown-menu>
-            <li class="menu-text organization-name">
-              <% if current_organization.present?  %>
-                <%= current_organization.name %>
-              <% elsif current_user.organization.present?  %>
-                <%= current_user.organization.name %>
-              <% else %>
-                <span>DiaperBase</span>
-              <% end %>
-            </li>
-            <li><%= navigation_link_to('Dashboard', dashboard_path) %></li>
-            <li><%= navigation_link_to('Donations', donations_path) %></li>
-            <li><%= navigation_link_to('Distributions', distributions_path) %></li>
+<div class="top-bar" id="main-menu">
+  <div class="top-bar-title">
+    <% if user_signed_in? %>
+      <% if current_organization && current_organization.logo.present? %>
+        <%= image_tag current_organization.logo.url, alt: current_organization.name, id: "logo" %>
+      <% else %>
+        <%= image_tag "DiaperBase-Logo.png", alt: "DiaperBase Logo", id: "logo" %>
```

Comment:
I've pushed up the change I just showed in that snippet. It's stable.

LG to close?